seigniorage.tex

```diff
@@ -150,22 +150,36 @@ \subsection{Reward Allocation and Eligibility}
 \bar{I}=\{i\in I \mid \mathcal{L}(i) \in V_i\}
 \end{equation}
 
-We introduce the following function to calculate the total supply of CLX
+We introduce the function
 
 \begin{equation}
 Q(i) = Q_0 (1+\sigma)^{i/\text{ticks\_per\_year}}
 \end{equation}
 
-where $Q_0$ is the initial supply, $\sigma$ is the yearly seiginorage rate and $\text{ticks\_per\_year} = 365\times 24\times 60 \times 60 \times 1000 = 31,\!536,\!000,\!000$.
+to model how the CLX supply would evolve under \emph{ideal conditions}\footnote{The function $Q$ does not dictate the exact size of the supply, but rather, used to set an upper limit to how much it can be at a given tick.}. Here, $Q_0$ is the initial supply, $\sigma$ is the yearly seiginorage rate and $\text{ticks\_per\_year} = 365\times 24\times 60 \times 60 \times 1000 = 31,\!536,\!000,\!000$. However, CLX supply will most likely not grow ideally, due to slashings and rounds which are not rewarded. Our goal is to make the \emph{real} rate of seigniorage stay constant, so we need to base round rewards on the current supply, rather than the ideal.
 
+Instead of forcing the protocol to keep track of changes to supply on a block-by-block basis, we take the final supply of the previous era as a benchmark, to calculate how much the round reward should be for a given era, throughout which it stays constant. This greatly reduces the complexity of what needs to be implemented without sacrificing anything that is crucial for seigniorage to work. We also assume that there is an optimal round exponent, $n^\ast$ , which all the validators should operate at, in ideal conditions. This allows us to derive the seigniorage rate for an optimal round
```

Comment:
I don't understand what it means to "assume an optimum" here. What is being optimized? And if something is, indeed, being optimized, how does the system figure out what it is?

Reply:
We are trying to maximize the combination of throughput (gas/s) and user experience (lower time-to-finality = better UX). *We* do the optimization beforehand by doing benchmarks, and assign `optimal_round_exponent` as a platform parameter.